README.md

```diff
@@ -4,7 +4,9 @@ README.md
 
 This is a simple web-based Tic Tac Toe game that allows two players to compete against each other. 
 
-![image](https://user-images.githubusercontent.com/66154908/236675164-d66403be-460f-43a2-b696-00cd0a1ea1d1.png)
+
+
```

Comment:
can you delete the extra new line

Reply:
If not then its ok. As you are new to open source I will directly approve the changes and merge the PR

Reply:
okay ill work to remove the extra lines